packages/@aws-cdk/aws-cognito/lib/user-pool-email.ts

```diff
@@ -22,14 +22,14 @@ export interface UserPoolSESOptions {
    * An optional name that should be used as the sender's name
    * along with the email.
    *
-   * @default - no name
+   * @default no name
```

Comment:
There's been some confusion around the use of these `-`s: the convention we follow is to use the `-` if the text that follows the `@default` is a description of the value (eg `no name`), and to not use the `-` if the string that follows is a valid value (eg `1`, `true`). Same comment everywhere else.